src/Navigation/Navigation.Xamarin.Forms/Service/NavigationService.cache.cs

```diff
@@ -22,20 +23,49 @@ public void ClearCache()
             PageCache.Clear();
         }
 
-        private void CreateCachedPage(
+
+        private static TPage AddOrUpdateContentCache<TPage>(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'AddOrUpdateContentCache'.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5575746)